data/maps/blocks.asm

```diff
@@ -444,9 +444,9 @@ SECTION "GoldenrodMagnetTrainStation_BlockData", ROMX
 GoldenrodMagnetTrainStation_BlockData:
 	INCBIN "maps/GoldenrodMagnetTrainStation.ablk.lz"
 
-SECTION "Daycare_BlockData", ROMX
-Daycare_BlockData:
-	INCBIN "maps/Daycare.ablk.lz"
+SECTION "Daycare_BlockData", ROMX ; TODO: rename to DayCare_BlockData
```

Comment:
I'll wait for this TODO to merge, then.

Reply:
How did I forget about this it's literally a sed lol

Reply:
Oh right because it changes the md5 hash